internal/maintenance/periodic_job_enqueuer.go

```diff
@@ -341,28 +338,13 @@ func (s *PeriodicJobEnqueuer) insertBatch(ctx context.Context, insertParamsMany
 	}
 	defer tx.Rollback(ctx)
 
-	queues := make([]string, 0, len(insertParamsMany))
-
 	if len(insertParamsMany) > 0 {
-		results, err := tx.JobInsertFastMany(ctx, insertParamsMany)
+		_, err := s.Config.Insert(ctx, tx, insertParamsMany)
 		if err != nil {
 			s.Logger.ErrorContext(ctx, s.Name+": Error inserting periodic jobs",
 				"error", err.Error(), "num_jobs", len(insertParamsMany))
 			return
 		}
-		for _, result := range results {
-			if !result.UniqueSkippedAsDuplicate {
-				queues = append(queues, result.Job.Queue)
-			}
-		}
-	}
-
-	if len(queues) > 0 {
-		if err := s.Config.NotifyInsert(ctx, tx, queues); err != nil {
-			s.Logger.ErrorContext(ctx, s.Name+": Error notifying insert", "error", err.Error())
-			return
-		}
-		s.TestSignals.NotifiedQueues.Signal(queues)
```

Comment:
I like that this change ended up simplifying some things, like not needing to deal with insert notifications in the `PeriodicJobEnqueuer` anymore. Now it's all the same insert logic as the `Client`.